bentoml/deployment/aws_lambda/__init__.py

```diff
@@ -0,0 +1,376 @@
+                'MemorySize': memory_size,
+                'Events': {
+                    'Api': {
+                        'Type': 'Api',
+                        'Properties': {
+                            'Path': '/{}'.format(api_name),
+                            'Method': 'post',
+                        },
+                    }
+                },
+                'Policies': [{'S3ReadPolicy': {'BucketName': s3_bucket_name}}],
+            },
+        }
+
+    yaml.dump(sam_config, Path(template_file_path))
+
+    # We add Outputs section separately, because the value should not have 'around !Sub
+
+    with open(os.path.join(project_dir, 'template.yaml'), 'a') as f:
+        f.write(
+            """\
+Outputs:
+    EndpointUrl:
+        Value: !Sub "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod"
+        Description: URL for endpoint
+"""
+        )
+    return template_file_path
+
+
+class AwsLambdaDeploymentOperator(DeploymentOperatorBase):
+    def apply(self, deployment_pb, yatai_service, prev_deployment):
+        try:
+            ensure_sam_available_or_raise()
+            ensure_docker_available_or_raise()
+            deployment_spec = deployment_pb.spec
+
+            bento_pb = yatai_service.GetBento(
+                GetBentoRequest(
+                    bento_name=deployment_spec.bento_name,
+                    bento_version=deployment_spec.bento_version,
+                )
+            )
+            if bento_pb.bento.uri.type not in (BentoUri.LOCAL, BentoUri.S3):
+                raise BentoMLException(
+                    'BentoML currently not support {} repository'.format(
+                        bento_pb.bento.uri.type
+                    )
+                )
+
+            return self._apply(
+                deployment_pb, bento_pb, yatai_service, bento_pb.bento.uri.uri
+            )
+        except BentoMLException as error:
+            return ApplyDeploymentResponse(status=exception_to_return_status(error))
+
+    def _apply(self, deployment_pb, bento_pb, yatai_service, bento_path):
+        if loader._is_remote_path(bento_path):
+            with loader._resolve_remote_bundle_path(bento_path) as local_path:
+                return self._apply(deployment_pb, bento_pb, yatai_service, local_path)
+
+        try:
+            deployment_spec = deployment_pb.spec
+            aws_config = deployment_spec.aws_lambda_operator_config
+            bento_service_metadata = bento_pb.bento.bento_service_metadata
+
+            # TODO
+            python_runtime = 'python3.7'
+            bento_service_py_version = version.parse(
+                bento_service_metadata.env.python_version
+            )
+            if bento_service_py_version < version.parse('3.0.0'):
+                raise BentoMLException(
+                    'Python 2 is not supported for Lambda Deployment'
+                )
+            elif (
+                version.parse('3.6.0')
```

Comment:
let's get rid of the "packaging" dependency in setup.py, this can be done easily without that library too - 

we got the bento_service_py_version by 
```
PYTHON_VERSION = "{major}.{minor}.{micro}".format(
    major=version_info.major, minor=version_info.minor, micro=version_info.micro
)
```
https://github.com/bentoml/BentoML/blob/master/bentoml/service_env.py#L26

major, minor, _  = bento_service_py_version.split('.')

if major != 3:
   raise...

python_runtime = 'python{}.{}'.format(major, minor)